### ISSUE TITLE
Feeds only at project not collection level

### ISSUE THREAD

**Issue by phatcher** [MicrosoftDocs/vsts-docs#344]
Docs say "Feeds exist at the project collection level to allow component sharing across projects" but I can't create a feed unless its associated with a team project.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 803b8ea5-ce05-6008-176d-7377a5c4dbf1
* Version Independent ID: a1e195c6-5dd0-5e5f-ce89-3dea4ed941f7
* Content: [Give teams and builds access to feeds](https://docs.microsoft.com/en-us/vsts/package/feeds/common-identities)
* Content Source: [docs/package/feeds/common-identities.md](https://github.com/MicrosoftDocs/vsts-docs/blob/live/docs/package/feeds/common-identities.md)
* Service: **unspecified**
* GitHub Login: @douge
* Microsoft Alias: **amullans**

**Comment by alexmullans** [comment]
That's right. Feeds exist at the project collection (account) level, but we do use your current project context when creating a feed to set up some of the permissions.

/cc @steved0x

**Comment by phatcher** [comment]
@alexmullans Ah, I was misled by the UI, because Packages are on the same menu as Builds and Release I presumed that they were at the Project level, but I can see now that the url is "https://<<account>.pkgs.visualstudio.com/_packaging/<feed>/nuget/v3/index.json"

**Comment by alexmullans** [comment]
Yeah, all of the packaging UX is at the project level since that's easiest in VSTS' current navigation paradigm. Sorry for the confusion.

**Comment by phatcher** [comment]
NP I can change my feed name now to reflect the collection not the project :-)

What would be nice now are VSTS build triggers based on feed/package changes - had this about 5 years ago in TeamCity and used it to keep ~60 projects all up to date against the core libraries

**Comment by alexmullans** [comment]
We've talked about such a feature with the build team, but I don't know where it is on the backlog. @chrisrpatterson any thoughts? We have the events flowing from the packaging side.

**Comment by phatcher** [comment]
If you have the events is it possible to write a custom Task to do this or does it have to be wired in higher up?

If the event can be published may be I could have an Azure function that then triggered the build?
